e2e/scripts/actions/from_uri.test

```diff
@@ -0,0 +1,38 @@
+#!/singularity/test
+# run from URI
+
+source common.func
+
+image=`get-test-image`
+
+runscript="${SOURCEDIR}/e2e/testdata/runscript.sh"
+option="--bind ${runscript}:/.singularity.d/runscript"
+size=`stat -c %s ${runscript}`
+
+# Run from supported URI's and check the runscript call works
+expect-success "RunFromDockerOK"    singularity run ${option} docker://busybox:latest ${size}
+expect-success "RunFromLibraryOK"   singularity run ${option} library://busybox:latest ${size}
+expect-success "RunFromShubOK"      singularity run ${option} shub://singularityhub/busybox ${size}
+expect-failure "RunFromDockerKO"    singularity run ${option} docker://busybox:latest 0
+expect-failure "RunFromLibraryKO"   singularity run ${option} library://busybox:latest 0
+expect-failure "RunFromShubKO"      singularity run ${option} shub://singularityhub/busybox 0
+
+# exec from a supported URI's and check the exit code
```

Comment:
It would be important to check the output and the success from some of these commands

Reply:
Will look